app/assets/stylesheets/base/_variables.sass

```diff
@@ -15,6 +16,9 @@ $application-link-hover-color: $screensmart-yellow
 $breakpoint-mobile: 500px
 $breakpoint-mobile-v: 500px
 
+// typography
+$screensmart-font-size: 1.2rem
```

Comment:
Van mij hoeft de `screensmart-` prefix niet, het is wel duidelijk in welke applicatie we zitten ;)